x/emissions/types/params.go

```diff
@@ -568,6 +558,22 @@ func validateMaxStringLength(_ uint64) error {
 	return nil
 }
 
+func validateInitialRegretQuantile(i alloraMath.Dec) error {
```

Comment:
can we get some nice function header comments, like every other `validate` function in this file has 

AI will autogenerate this for you

ought to add a linter rule to throw if any function in the repo doesn't have function header